playground/android/app/build.gradle

```diff
@@ -1,19 +1,12 @@
 apply plugin: "com.android.application"
+apply plugin: "com.facebook.react"
 
-project.ext.react = [
-        root     : "../../../",
-        entryFile: "index.js",
-        cliPath : "node_modules/react-native/cli.js",
-        bundleAssetName: "index.android.bundle",
-        bundleInAlpha: true,
-        bundleInBeta: true,
-        enableHermes: true,
-        hermesFlagsDebug:['-Xes6-proxy','-output-source-map'],
-        hermesFlagsRelease:['-output-source-map'],
-        hermesCommand: "../../../node_modules/react-native/sdks/hermesc/%OS-BIN%/hermesc",
-]
-
-apply from: "../../../node_modules/react-native/react.gradle"
+react {
+     root = file("../../../")
```

Comment:
is that all we need?

What about?

```
react = {
    root = file("../../../")
    entryFile = file("index.js")
    cliPath = file("node_modules/react-native/cli.js")
    bundleAssetName = "index.android.bundle"
    bundleInAlpha = true
    bundleInBeta = true
    enableHermes = true
    hermesFlagsDebug = ['-Xes6-proxy','-output-source-map']
    hermesFlagsRelease = ['-output-source-map']
    hermesCommand = file("../../../node_modules/react-native/sdks/hermesc/%OS-BIN%/hermesc")
}
```
Do we not care about the other values that used to be there?

Reply:
I think those are RN's defaults